ChatDemo-UI3.0/ChatDemo-UI3.0/en.lproj/Localizable.strings

```diff
@@ -316,12 +316,12 @@
 "setting.inputName" = "Please input nickname";
 "setting.saveName" = "Save Nickname";
 "setting.namenotempty" = "Name cannot be empty";
-"setting.personalInfo" = "Profile";
+"setting.personalInfo" = "User Profile";
 "setting.setBitrate" = "Set Bitrate";
 "setting.setBitrateTips" = "Set Bitrate should be 150-1000";
 "setting.personalInfoUpload" = "Profile Image";
 "setting.edittips" = "After setting this nickname, chat with the iOS client demo project, iOS will display this nickname is not a EaseMob ID, if the other party to use the Android client this setting is not effective";
-"setting.iospushname" = "iOS push nickname";
+"setting.iospushname" = "APNs Display Name";
```

Comment:
Since you changed APNS settings to Push Notification, I assume Notification Display Name is better here, APNS is not necessarily known by everyone